[4/5]Testing jit module in flatbuffer in Python.

qihqi opened the pull request:
Summary: Make temporary python bindings for flatbuffer to test ScriptModule save / load.

Test Plan: unittest

Differential Revision: D34968080

iseeyuan commented on setup.py:
```diff
@@ -824,7 +824,16 @@ def make_relative_rpath_args(path):
                   include_dirs=[],
                   library_dirs=library_dirs,
                   extra_link_args=extra_link_args + main_link_args + make_relative_rpath_args('lib'))
+    C_flatbuffer = Extension("torch._C_flatbuffer",
```
It's a pretty high-level and user-facing extension. Why not adding `_flatbuffer` to `torch._C', so we don't have to add a new extension here?

qihqi replied:
`//caffe2:torch` cannot depend on flatbuffer without breaking some downstreams. So the use of flatbuffer features for them is opt-in. Therefore, it has to be separate .so to be generated, and thus different names.

qihqi committed:
[4/5]Testing jit module in flatbuffer in Python. (#74387)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/74387

Make temporary python bindings for flatbuffer to test ScriptModule save / load.

Test Plan: unittest

Reviewed By: iseeyuan

Differential Revision: D34968080

fbshipit-source-id: 7183f514821573a7543531465f53e115467a9daa
